odl/tomo/backends/skimage_radon.py

```diff
@@ -11,13 +11,19 @@
 from __future__ import division
 
 import numpy as np
+import warnings
 
-from odl.discr import uniform_discr_frompartition, uniform_partition
+from odl.tomo import Parallel2dGeometry, Geometry
```

Comment:
Missed this, you mean to import directly via `odl.discr.discr_space`? Here we are in `odl.tomo.backends`?

Reply:
No, I mean the `tomo` imports. Maybe this is based on me misunderstanding the Python import system, but it seems to me that it would be easier to break import cycles when import paths are deeper.
The above version is actually a cyclic import, because it says "import the `odl.tomo`, subpackage, *which includes myself*, and give me its attributes `Parallel2dGeometry` and `Geometry`.

Reply:
I guess it makes sense not to import from your parent indeed. Wonder what happens though at import-time: if the parent is already imported, then Python may not re-import the parent again? :thinking: